Mvc.sln

```diff
@@ -919,6 +906,30 @@ Global
 		{FCFE6024-2720-49B4-8257-9DBC6114F0F1}.Release|Mixed Platforms.Build.0 = Release|Any CPU
 		{FCFE6024-2720-49B4-8257-9DBC6114F0F1}.Release|x86.ActiveCfg = Release|Any CPU
 		{FCFE6024-2720-49B4-8257-9DBC6114F0F1}.Release|x86.Build.0 = Release|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Debug|Mixed Platforms.ActiveCfg = Debug|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Debug|Mixed Platforms.Build.0 = Debug|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Debug|x86.Build.0 = Debug|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Release|Any CPU.Build.0 = Release|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Release|Mixed Platforms.ActiveCfg = Release|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Release|Mixed Platforms.Build.0 = Release|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Release|x86.ActiveCfg = Release|Any CPU
+		{72AF67AF-D523-4321-AF0F-2287117E67A0}.Release|x86.Build.0 = Release|Any CPU
```

Comment:
I had a merge conflict in Mvc.sln, apparently I messed it up and this is the result.